resources/testProfileData.example.ts

```diff
@@ -2,10 +2,10 @@ import { IProfile } from "@brightside/imperative";
 
 export const profile: IProfile = {
     type : "zosmf",
-    host: "",
-    port: 0,
-    user: "",
-    password: "",
+    host: "http://tvt4119.svl.ibm.com",
+    port: 443,
+    user: "zoweew",
+    password: "waziusr",
```

Comment:
This file should not be changed, it's an example. If you want to run integration tests you need to create testProfileData.ts. Here's README with more info https://github.com/phaumer/vscode-extension-for-zowe/tree/master/docs.
Also, don't put usernames and password in source code.